Indicators/ChandeVariableIndexDynamicAverage.cs

```diff
@@ -0,0 +1,96 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// This indicator computes the n-period adaptive weighted moving average indicator.
+    /// VIDYAi = Pricei x F x ABS(CMOi) + VIDYAi-1 x (1 - F x ABS(CMOi))
+    /// where:
+    /// VIDYAi - is the value of the current period.
+    /// Pricei - is the source price of the period being calculated.
+    /// F = 2/(Period_EMA+1) - is a smoothing factor.
+    /// ABS(CMOi) - is the absolute current value of CMO.
+    /// VIDYAi-1 - is the value of the period immediately preceding the period being calculated.
+    /// </summary>
+    public class ChandeVariableIndexDynamicAverage : WindowIndicator<IndicatorDataPoint>, IIndicatorWarmUpPeriodProvider
```

Comment:
Could you please rename it to `VariableIndexDynamicAverage`?

Reply:
Sure👍